libs/partners/cohere/tests/integration_tests/test_rerank.py

```diff
@@ -0,0 +1,16 @@
+"""Test Cohere reranks."""
+from langchain_core.documents import Document
+
+from langchain_cohere import CohereRerank
+
+
+def test_langchain_cohere_rerank_documents() -> None:
+    """Test cohere rerank."""
+    rerank = CohereRerank()
+    test_documents = [
+        Document(page_content="This is a test document."),
+        Document(page_content="Another test document."),
+    ]
+    test_query = "Test query"
+    results = rerank.rerank(test_documents, test_query)
+    assert len(results) == 2
```

Comment:
How about creating a list of 4 (or more) docs and asserting the length of the results to be equal to 3 (default `n`)?

Reply:
will follow up @mspronesti thanks!

Reply:
You can click on "commit suggestion" if that's faster :)

```suggestion
    test_documents = [
        Document(page_content="This is a test document."),
        Document(page_content="Another test document."),
        Document(page_content="Yet Another test document."),
        Document(page_content="Fourth test document."),
    ]
    test_query = "Test query"
    results = rerank.rerank(test_documents, test_query)
    assert len(results) == 3
```